hosts/pds3.py

```diff
@@ -0,0 +1,113 @@
+################################################################################
+# oops/inst/pds3.py
+#
+# Utility functions for finding and loading PDS3 labels in host modules.
+#
+################################################################################
+import os
+import pdsparser
+from pathlib import Path
+
+#===============================================================================
+class PDS3(object):
+    """An instance-free class for PDS3 label management methods."""
+
+    #===========================================================================
+    @staticmethod
+    def clean_label(lines):
```

Comment:
Because the content of clean_label is going to depend on the quirks of each data set, this makes me a bit nervous. I could imagine a situation where code that cleans up one label would break another. Here's a better way: Add a new input "cleaner=None" to get_label. Inside get_label,
```
if cleaner is not None:
    lines = cleaner(lines)
```
So now "cleaner" is an optional function that you can define inside any of the instrument modules. If you pass it to get_label, that function will be used to edit the lines after they are read and before they are parsed. You would then need to define tiny cleaner functions for Cassini VIMS and UVIS, but nothing instrument-specific would show up here.

If there were a set of very common PDS3 syntax errors, then it would make a bit more sense to have a dedicated function like this one (in addition to, not instead of, the instrument-specific functionns). For example, _maybe_ it's worth a general check to make sure "N/A" is always surrounded by quotes, because I have seen that error before. ((However, if we did that test by default, we probably wouldn't make the assumptions about the adjacent parentheses that appear inside these VIMS tests.) That's really a judgment call.

Reply:
My original intent was to hide all that stuff in this module, but you're right that I can't guarantee that some of these corrections would not break label from some other host.  So I did as you suggest, but I added the ability to have built-in standard cleaners that get called by default along with any custom cleaners provided by the host module.   Also, I have the cleaners operating on a single line to simplify the host code.

I have not tested the custom cleaners in order to .